closures.py

```diff
@@ -0,0 +1,8 @@
+# your code goes here
+def multiplier_of(n:int) -> object:
```

Comment:
```suggestion
from typing import Callable

def multiplier_of(n:int) -> Callable:
```

most things that support the `()` operators are classified as [Callables](https://docs.python.org/3/library/functions.html#callable)

While object is technically correct, and doing something like `print(isinstance(multiplywith5, object))` would return `True`, the object type itself is too vague to help anyone understand what kind of value the code would be returning. Since even something like `print(isinstance(1, object))` would return `True` as all objects are instances of `object`. [Most type checkers will instantly object to the use of `object` for type hints](https://mypy.readthedocs.io/en/latest/dynamic_typing.html#any-vs-object), as that implies that the value being returned is an instance of the root `object` class which will almost never be the case. If you want to create a function that does not have a clear return type, and can return any type of value,  [use `typing.Any`](https://docs.python.org/3/library/typing.html#the-any-type) instead

Reply:
Thanks! I will keep that in mind.